### ISSUE TITLE
`source: not a directory` error on new terminal start when `.env` file present in project root

### ISSUE THREAD

**Issue by taylorthurlow** [zed-industries/zed#15570]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Opening a new terminal window is trying to execute an `activate` script in my project which doesn't exist. This is what appears in the terminal after opening:

```
source /Users/taylor/mydir/.env/bin/activate
taylor@cintra:mydir/ % source /Users/taylor/mydir/.env/bin/activate
source: not a directory: /Users/taylor/mydir/.env/bin/activate
```

My project directory is `/Users/taylor/mydir`. The source error only appears when I have a `.env` file in my project root directory. This did not happen on Zed Preview v0.146.3.

### Environment

Zed: v0.147.0 (Zed Preview)
OS: macOS 14.5.0
Memory: 16 GiB
Architecture: aarch64

**Comment by notpeter** [comment]
I can reproduce this as well.  As a temporary workaround you can add this to your settings.
```json
  "detect_venv": {
    "on": {
      "directories": ["env", ".venv", "venv"]
    }
  },
```

**Comment by notpeter** [comment]
I did the git bisect and it broke at 583b6235fb9eda97da82adbfe1ccc588015c0385 which is:
- https://github.com/zed-industries/zed/pull/15321

@ConradIrwin

**Comment by failable** [comment]
Same issue after updated to latest preview version.

**Comment by plugd-in** [comment]
I think this issue is just a matter of removing venv stuff from `crates/project/src/terminals.rs`. If it works, I'll add the commit to the tip of my fork.

**Comment by ConradIrwin** [comment]
I think we can improve this by updating the check to ensure the venv is a directory.

in the meantime you can disable the venv stuff by setting it to off in settings

**Comment by taylorthurlow** [comment]
Can confirm that this is fixed in `v0.149.1-pre` (https://github.com/zed-industries/zed/pull/15989).
